library/datasort.c

```diff
@@ -817,6 +894,10 @@ static struct datasort_chunk *datasort_merge_get_smallest(struct datasort_cfg *d
 	list_for_each_entry(chunk, &dcfg->sorted_chunks, list) {
 		assert(chunk->merge_count <= chunk->count);
 
+		// Skip removed records
+		while (chunk->merge_count != chunk->count && chunk->index[chunk->merge_count].flags & BLOB_DISK_CTL_REMOVE)
```

Comment:
Code style: line exceed 120 symbols, try not to add new lines > than 120 symbols length.

Reply:
Tiny optimization: we can do this loop only if chunk is base_view, otherwise we can check this flag on iteration, and not to add removed keys to sorting.

Reply:
Maybe mandatory optimization. And maybe we will reuse it later. I think worth leave it.